ambry-api/src/main/java/com.github.ambry/messageformat/BlobProperties.java

```diff
@@ -23,17 +23,23 @@
  */
 public class BlobProperties {
 
+  public static final short ACCOUNTID_CONTAINERID_DEFAULT_VALUE = -1;
```

Comment:
Yes I get that. But both goes hand in hand and the default is same. By hand in hand, I mean, when we plan to enable accountId, we would enable containerId too.